e2e_test/batch/aggregate/scalar_as_agg.slt.part

```diff
@@ -0,0 +1,9 @@
+query T
+select aggregate:array_sum(v) from (values (3), (2), (1)) as t(v);
```

Comment:
If technically possible, I personally prefer `array_sum(v)` than `aggregate:`

Reply:
I prefer an explicit annotation of the scalar->aggregate usage, otherwise both user and implementor would be confused on whether a function is called as scalar or aggregate. 🤔